src/main/java/org/group23/NumericalQuestion.java

```diff
@@ -0,0 +1,36 @@
+package org.group23;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class NumericalQuestion extends Question {
```

Comment:
We need to implement a question "asking for a number within a range": need bounds for the minimum and maximum (both inclusive).

Reply:
I implemented it